features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1590,11 +1589,11 @@ CREATE TABLE IDN_USER_FUNCTIONALITY_PROPERTY (
 )
 /
 
-CREATE TABLE IF NOT EXISTS IDN_CORS_ORIGIN (
-    ID                INT           NOT NULL AUTO_INCREMENT,
-    TENANT_ID         INT           NOT NULL,
-    ORIGIN            VARCHAR(2048) NOT NULL,
-    UUID              CHAR(36)      NOT NULL,
+CREATE TABLE IDN_CORS_ORIGIN (
+    ID        INT           NOT NULL AUTO_INCREMENT,
```

Comment:
Dont think this is a valid Syntax in DB2

Reply:
I'll fix it through a different PR.

Reply:
https://github.com/wso2/carbon-identity-framework/pull/3126

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3126 @ashensw